v3/src/components/graph/components/graph.tsx

```diff
@@ -39,6 +38,7 @@ import {mstReaction} from "../../../utilities/mst-reaction"
 import {onAnyAction} from "../../../utilities/mst-utils"
 import {IPixiPointsRef} from "../utilities/pixi-points"
 import {Adornments} from "../adornments/adornments"
+import { DotPlotDots } from "./dotplotdots"
```

Comment:
May want to move this import up with the other "plot" components for consistency.